views/editor/bramble.html

```diff
@@ -117,10 +117,13 @@
 
     <div id="spinner-container">
         <div class="thimble-spinner"></div>
-        <div class="loading-message message">{{ gettext("loadingProject") }}</div>
+        <div class="loading-message message">{{ gettext("loadingProject") }}
+	<p>{{ gettext("loading problem") }} <button type="button"><a href="https://support.mozilla.org/products/webmaker/thimble">{{ gettext("get help here") }}</button></a></p>
```

Comment:
`loading problem` is probably not the best name for the key (esp. since it has a space in it). Maybe something like `loadingProjectSupportSuggestion`? You would also need an entry for it somewhere [here](https://github.com/mozilla/thimble.mozilla.org/blob/master/locales/en-US/messages.properties#L75-L80). Add an entry with whatever text you think might be appropriate for now and we can suggest changes. Also, I think you should enter the entire contents of the paragraph as one localization string i.e. something like `loadingProjectSupportSuggestion=Blah blah blah <a href="...">Thimble help</a>.`